core/commands/generate_product_repositories_command.rb

```diff
@@ -224,10 +220,69 @@ def parse_maxscale_ci_deb_repository(config)
     )
   end
 
+  def get_maxscale_repository_name_for_docker(base_url, username, password)
+    uri_with_names_repos = base_url + '/v2/_catalog'
+    begin
+      doc = Nokogiri::HTML(open(uri_with_names_repos, http_basic_authentication: [username, password]))
+      names = JSON.parse(doc.css('p').children.to_s)
+      names.dig('repositories').first
```

Comment:
Why only first name is getting here? We could probably have several products inside this registry.

For now I suggest to get tags for mariadb/maxscale-ci directly without this step.